src/auth/index.ts

```diff
@@ -21,29 +24,77 @@ router.post("/login", async function(req, res) {
     where: {
       email: req.body.email
     }
-  }).catch(err => {
-    console.log(err);
-    return [];
-  });
-
-  if (queryResult.length > 0 && queryResult[0].password === req.body.password) {
-    const token = jwt.sign(
-      {
-        email: queryResult[0].email,
-        userId: queryResult[0].userId
-      },
-      "secret",
-      {
-        expiresIn: "30s"
-      }
-    );
-    res.json({
-      token,
-      message: "You are logged in! ✅"
+  })
+    .then(result => {
+      return result[0];
+    })
+    .catch(err => {
+      res.status(404).json({ message: err.message });
+      return null;
     });
+
+  if (queryResult) {
+    bcrypt
+      .compare(req.body.password, queryResult.password)
+      .then(isPassword => {
+        if (isPassword) {
+          const token = createTkn(queryResult);
+          res.status(200).json({
+            message: "You are logged in ✅",
+            token
+          });
+        } else {
+          res.status(401).json({
+            message: "Auth failed ⛔️"
+          });
+        }
+      })
+      .catch(err => {
+        res.status(404).json({ message: err.message });
+      });
   } else {
     res.status(401).json({
-      message: "Authorization Failed ⛔️"
+      message: "Failed to login"
     });
   }
 });
+
+router.post("/signin", async function(req, res) {
+  await validateLogin(req.body).catch(() => {
+    res.status(401).json({
+      message: "Authorization Faild: Invalid information ⛔️"
+    });
+  });
+
+  await User.findAll({
```

Comment:
@Alexandria You might not need this `await` since there is no code being executed after this call.